Quadriga/src/main/webapp/WEB-INF/views/failure_transform.jsp

```diff
@@ -0,0 +1,13 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>Insert title here</title>
+</head>
+<body>
+
+	<h1>Transformation is unsuccessful</h1>
```

Comment:
Take a look at how we use tiles again. 
Also, there should be at least a heading and a paragraph.